ocean_lib/models/test/test_datatoken_enterprise.py

```diff
@@ -87,24 +77,10 @@ def test_buy_from_dispenser_and_order(
         {"from": publisher_wallet},
     )
 
-    provider_fee_address = publisher_wallet.address
-    provider_fee_token = DAI.address
-    provider_fee_amount = 0
-    provider_data = json.dumps({"timeout": 0}, separators=(",", ":"))
     valid_until = 1958133628  # 2032
```

Comment:
in non-unit tests, it is exactly what we do, see e.g. test_reuse_order_fees, where we create the timestamp on the fly and send it towards the model class. For unit testing purposes we added a timestamp directly.

Would you like:
- the function itself to take in a datetime? -> I would not recommend it, because I'd prefer we stay closer to the API
- create this known `valid_until` value in tests, on the fly, instead of directly writing it?

Reply:
The function itself should still take the Unix time, that's ok. Just compute it here in the test using datetime. Should be a short single line.